build-system/npm-publish/write-package-files.js

```diff
@@ -197,6 +197,19 @@ async function writeReactJs() {
   }
 }
 
+/**
+ * todo(kvchari): temporarily copy styles to root of each package to support importing styles from root
+ * Remove when we begin properly proxying style imports.
+ * See issue from more information:
+ * @return {Promise<void>}
+ */
+async function copyCssToRoot() {
+  const extDir = path.join('extension', extension, '1.0');
+  const preactCssDist = path.join(extDir, 'dist', 'styles.css');
+  const preactCssRoot = path.join(extDir, 'styles.css');
+  await copyFile(preactCssDist, preactCssRoot);
```

Comment:
Can you wrap this in a try/catch similar to `writeReactJs`?

Reply:
done